### PR TITLE
tests: remove support for ubuntu 23.10

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Ubuntu mantic is already EoL

Ubuntu-no-lts will be in github workflow but no executions will be done. This is done to avoid removing it and force everyone to rebase master.

### CODE REVIEW

**Review by andrewphelpsj — APPROVED** [review]
thanks!

File: tests/lib/tools/tests.session [hunk 1]
```diff
@@ -266,6 +266,7 @@ main() {
 		--pipe \
 		--service-type simple \
 		--setenv TERM=xterm-256color \
+		--setenv XDG_RUNTIME_DIR=$XDG_RUNTIME_DIR \
```
Comment (ZeyadYasser) [inline tests/lib/tools/tests.session:269]: Is this relevant to the PR?

**Review by ZeyadYasser — APPROVED** [review]
Thanks!

File: tests/lib/tools/tests.session (hunk 1 above)
Comment (sergiocazzolato) [reply tests/lib/tools/tests.session:269]: no, I'll remove it. Thanks for the review
